Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Impute"
 uuid = "f7bf1975-0170-51b9-8c5f-a992d46b9575"
 authors = ["Invenia Technical Computing"]
-version = "0.6.3"
+version = "0.6.4"
```

Comment:
Not sure if considered a patch or a feature

Reply:
No, I think this is just a bug / edge cases. The corresponding `impute` method already does this.